docs/release-notes/release-notes-0.9.7.md

```diff
@@ -0,0 +1,162 @@
+
+`a3e1ba3` Default 3rd argument of gettxout should be True
+
+### New script to calculate fidelity bond values
+
+This new script is intended to help users make more informed choices about fidelity bonds. It gives data on the value of the bond dependent on locktime and amount, and optionally can compare with existing bonds in the orderbook (using a json export of the latter).
+
+* `72bf447` Script: Add bond-calculator.py
+
+### New display of 'available' balance
+
+This new feature in the UI (both CLI and Qt) displays tags 'FROZEN', 'PENDING', 'LOCKED' to indicate the status of different utxos (with 'PENDING' meaning unconfirmed). Available balance is distinguished from total balance with parentheses.
+As you can see from this description already, the situation is substantially more complex for Joinmarket than for simpler wallets because there are at least 3 ways in which utxos' usability might be restricted, and this is not necessarily the same for all use cases (e.g. maker vs taker). Still, this UI addition gives the user a lot more information.
+This additional 'available balance' information is now also returned in the RPC API.
+
+* `015cb4a` If there is unavailable fund, display 2 balances: total balance and unlocked balance.
+* `ea6c34d` update yaml for new available_balance fields
+* `75a93df` fix available_balance
+
+We also had to update the json-ified wallet display to accommodate this:
+
+* `08581f8` Fixes json serialization of wallet display
+
+### RPC-API updates
+
+There continue to be several small updates to the API, most notable this time is the addition of schedule status to the `/session` endpoint.
+
+* `8ee6b39` Report offerlist in /session call if possible
+* `b2ab4db` add nickname field to session response
+* `2cb41fc` review: reaqd schedule from taker object, also `190d56e` `57d1476`
+* `ab2803e` fix: include schedule in session instead of status flag, and `521189a`
+* `77496cd` Disallow RPC directsend if coinjoin state enabled
+* `cde5cfb` feat: scheduler status flag in session
+* `3f358ed` Add options to scheduler API endpoint
+* `12bcbe1` docs: add flag to session response spec
+
+### Tor updates
+
+* `adc11a1` Add separate documentation for Tor configuration
+* `f8497f0` add missing text to tor.md
+* `d46a2c6` Use valid tor_root also for tor_install()
+* `5e9044e` Install Tor binaries under /usr/local for --docker-install
+
+### Qt changes
+
+We now allow access to the xpub information in the Qt interface (previously it was not displayed in the GUI).
+
+* `d95e279` Display the account xpub in QT interface
+* `e4f249a` Show QR code for xpub
+
+### Installation
+
+The installation now can automatically check signatures for libsodium and Tor; this can be switched off with `--no-gpg-validation`.
```

Comment:
```suggestion
The installation now will automatically check release GPG signatures for libsodium and Tor; this can be switched off with `--no-gpg-validation`.
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1335/commits/0af9ad40244981485cbdbfeceffaf726a0e2953d